articles/getting-started-with-ui-layouts-in-android/index.md

```diff
@@ -0,0 +1,140 @@
+This article will cover the different User Interface(UI) layouts used in android application developments.
+
+### Introduction
+The basic building block in the User Interface(UI) of an application is called a **View**. A view occupies a rectangular area in a mobile screen. A view contains a layout and a view group such as a `Button`, `ImageView`, `TextView`, and others.
```

Comment:
Please rephrase the third sentence. It seems inaccurate. `Button`, `ImageView`, and `TextView` are neither layouts nor view groups.